code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { formatDistanceToNow } from 'date-fns';
```

Comment:
curious to why you chosen { formatDistanceToNow }?
I used 
import { formatDistance } from 'date-fns';
Do you know the difference?